docs/src/examples/components/Menu/Usages/MenuExampleToolbar.shorthand.tsx

```diff
@@ -76,15 +76,91 @@ const items = [
     indicator: false,
     menu: {
       items: [
-        { key: '1', content: 'item1' },
+        {
+          key: '1',
+          content: 'item1',
+          icon: {
+            name: 'bookmark',
+            outline: true,
+          },
+        },
         {
           key: '2',
           content: 'item2',
+          icon: {
+            name: 'mark-as-unread',
+            outline: true,
+          },
         },
         {
           key: '3',
           content: 'item3',
-          menu: { items: [{ key: '1', content: 'item3.1' }, { key: '2', content: 'item3.2' }] },
+          icon: {
+            name: 'translation',
+            outline: true,
+          },
+        },
+        {
+          key: '4',
+          content: 'item3',
+          icon: {
+            name: 'trash-can',
+            outline: true,
+          },
+        },
+      ],
+    },
+  },
+  {
+    key: 'menuButton2',
+    icon: {
+      name: 'more',
+      outline: true,
+    },
+    accessibility: toolbarButtonBehavior,
+    'aria-label': 'More options',
+    indicator: false,
+    menu: {
+      items: [
+        {
+          key: '5',
+          content: 'item1',
+          icon: {
+            name: 'bookmark',
+            outline: true,
+          },
+        },
+        {
+          key: 'divider',
+          kind: 'divider',
+        },
+        {
+          key: '6',
+          content: 'item2',
+          icon: {
+            name: 'mark-as-unread',
+            outline: true,
+          },
+        },
+        {
+          key: '7',
+          content: 'item3',
+          icon: {
+            name: 'translation',
+            outline: true,
+          },
+        },
+        {
+          key: 'divider2',
+          kind: 'divider',
+        },
+        {
+          key: '8',
+          content: 'item3',
+          icon: {
+            name: 'trash-can',
+            outline: true,
+          },
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/53743735-2bafa200-3ea4-11e9-89cd-f32cdee8ef49.png)


These items looks quite strange, should we remove one of them?

Reply:
Sure. Removed one.